update lama export DS specs to be more accurate.

laithsakka opened the pull request:
Summary:
this PR https://github.com/pytorch/pytorch/pull/164075  enhances value range analysis discovering a problem in the current upper bounds for DS specs for lama.

This blocks landing the PR above.

laithsakka commented:
how do i re-run failed tests?

larryliu0820 reviewed (approved):
Review automatically exported from Phabricator review in Meta.

laithsakka committed:
update lama export DS specs to be more accurate. (#14737)

Summary:

this PR https://github.com/pytorch/pytorch/pull/164075  enhance value range analysis discovering a problem in the current upper bounds for DS specs for lama
full DS logs if someone want to understand why in details
https://www.internalfb.com/phabricator/paste/view/P1973006378

This blocks landing the PR above.

Reviewed By: angelayi, larryliu0820

Differential Revision: D83708583
